src/ui/ui-kit/BottomPanel/BottomPanel.tsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import cn from 'classnames';
+import { useSpring, animated } from '@react-spring/web';
+import * as styles from './styles.module.css';
+
+export const StickyBottomPanel = React.forwardRef(
+  (
+    {
+      className,
+      containerStyle,
+      backdropStyle,
+      children,
+      ...props
+    }: React.HTMLProps<HTMLDivElement> & {
+      containerStyle?: React.CSSProperties;
+      backdropStyle?: React.CSSProperties;
+    },
+    ref: React.Ref<HTMLDivElement>
+  ) => {
+    return (
+      <div className={styles.stickyBottomSheet} style={containerStyle}>
+        <div ref={ref} className={cn(className, styles.content)} {...props}>
+          {children}
+        </div>
+        <div
+          className={styles.backdrop}
+          style={{
+            top: 24,
+            bottom: -16,
+            left: -16,
+            right: -16,
+            ...backdropStyle,
+          }}
+        />
+      </div>
+    );
+  }
+);
+
+export const AnimatedBottomPanel = React.forwardRef(
```

Comment:
I wonder how this panel's focus should be handled when it's hidden. Let's discuss

Reply:
Removed